k8s/crds/agentendpoint.mdx

```diff
@@ -302,6 +302,316 @@ When the `spec.upstream.url` has an `https://` scheme, the client certificates w
 | `spec.clientCertificateRefs.name`      | `string` | yes      | none    | Name of a `Secret` resource to pull the client certificate data from                                                                       |
 | `spec.clientCertificateRefs.namespace` | `string` | no       | none    | Namespace of a `Secret` resource to pull the client certificate data from. When not supplied, the namespace of the `AgentEndpoint` is used |
 
+## Status Fields
+
+The `AgentEndpoint` resource includes status information that reflects the current state of the agent endpoint in the ngrok system.
+
+### `status.assignedURL`
+
+The actual URL assigned to the endpoint. This may differ from `spec.url` when using shorthand formats (like `https://` or empty).
+
+**Type:** `string`
+
+**Example:** `https://example-agent-endpoint.ngrok.app`
+
+### `status.trafficPolicy`
+
+Indicates which traffic policy is attached to the endpoint.
+
+**Type:** `string`
+
+**Possible Values:**
+- `inline` - Endpoint uses an inline traffic policy
+- `none` - No traffic policy is attached
+- `<policy-name>` - Name of the referenced NgrokTrafficPolicy resource
+
+**Example:** `inline`
+
+### `status.domainRef`
+
+Reference to the Domain resource associated with this agent endpoint.
+
+**Type:** `Object`
+
+**Fields:**
+- `name` - Name of the Domain resource
+- `namespace` - Namespace of the Domain resource
+
+**Example:**
+```yaml
+domainRef:
+  name: example-agent-endpoint-ngrok-app
+  namespace: default
+```
+
+### `status.conditions`
+
+Standard Kubernetes conditions that indicate the state of the agent endpoint.
+
+**Type:** `[]Condition`
+
+Each condition includes:
+- `type` - The condition type
+- `status` - `True`, `False`, or `Unknown`
+- `reason` - A programmatic identifier for the condition state
+- `message` - A human-readable description
+- `lastTransitionTime` - When the condition last changed
+- `observedGeneration` - The resource generation this condition applies to
+
+#### Condition Types
+
+##### `Ready`
+
+Indicates whether the agent endpoint is fully operational and ready to receive traffic.
+
+**Status Values:**
+- `True` - Agent endpoint is active and ready
+- `False` - Agent endpoint is not ready (see reason for details)
+
+**Common Reasons:**
+
+| Reason | Status | Meaning |
+|--------|--------|---------|
+| `EndpointActive` | `True` | Agent endpoint is successfully created and active |
+| `Reconciling` | `False` | Endpoint is being reconciled/updated |
```

Comment:
@jonstacks Does "reconciled" just mean "updated" in this context? Could we be a little more explicit about what's happening?

Reply:
It is used when the AgentEndpoint is reconciling and not yet ready. I think we're actually going to get rid of this Reason long term, so we'll just remove it here now.